spec/support/feature_helpers.rb

```diff
@@ -6,13 +6,18 @@ def stub_omniauth
       uid: "123456789",
       info: {
         first_name: "John",
-        last_name: "Smith"
+        last_name: "Smith",
+        email: "email@example.com"
       },
       credentials: {
         token_id: "123"
       }
     }
   end
+
+  def stub_session_create
+    allow(Session).to receive(:create).and_return(something: 'testing')
```

Comment:
Shouldn't this be something that has an `id` method to simulate how it's actually used?

```
OpenStruct.new(id: 1).id #=> 1
```

or, _I think_, which might be better if it works:

```
double(id: 1).id #=> 1
```

Reply:
nice one